mariannaFervenza/assignments/treesBFS/lc107/lc107.py

```diff
@@ -0,0 +1,111 @@
+# Problem Statement #
+
+# Given a binary tree, populate an array to represent its level-by-level traversal in reverse order, i.e., the lowest level comes first. You should populate the values of all nodes in each level from left to right in separate sub-arrays.
+
+
+# edge cases:
+
+# If the root is null then just return the empty array
```

Comment:
Nice overview here. Typing the solution twice can also be helpful in building muscle memory!